rocketpy/mathutils/function.py

```diff
@@ -1088,13 +1090,15 @@ def __call__(self, *args):
             evaluated at all points in the list and a list of floats will be
             returned. If the function is N-D, N arguments must be given, each
             one being an scalar or list.
+        filename : str | None, optional
+            The path the plot should be saved to. By default None, in which case the plot will be shown instead of saved. Supported file endings are: eps, jpg, jpeg, pdf, pgf, png, ps, raw, rgba, svg, svgz, tif, tiff and webp.
```

Comment:
In docstrings, please break your lines at around 80 characters. This makes it easier to read the docstrings in the source code and avoid errors with the documentation builder (Sphinx).

Reply:
Do you think it is worth to mention that these supported file formats are the same as the ones supported by matplotlib?

Reply:
> Do you think it is worth to mention that these supported file formats are the same as the ones supported by matplotlib?

Yes, I think that would be better. Do you think the explicit list of supported formats should still stay in there, or should we remove it? Otherwise, if matplotlib changes their supported formats, someone would have to update each docstring mentioning them